### PR TITLE
Exercise list user Storie 1

### PR DESCRIPTION
**Pull request by Mersag**

### CODE REVIEW

File: src/components/Card.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default function Card() {
+	return <div>cobra</div>;
+}
```
Comment (JessicaLoers) [inline src/components/Card.js]:
```suggestion
export default function Card({name, sanskritName, effects}) {
	return (
		<article>
			<h2>{name}</h2>
			<p>{sanskritName}</p>
			<ul>
				{effects.map((effect, index) => (
					<li key={index}>{effect}</li>
				))}
			</ul>
		</article>
	);
}
```

File: src/pages/index.js [hunk 2]
```diff
@@ -1,76 +1,27 @@
...
-			</section>
+			<h2>Exercises</h2>
+			<div>
+				<Card />
+				<ul>
+					{db.map(data => {
+						return <div key={data.id}>{data.name}</div>;
+					})}
+				</ul>
+			</div>
```
Comment (JessicaLoers) [inline src/pages/index.js]:
```suggestion
		{db.map(data => {
				return (
					<Card
						key={data.id}
						name={data.name}
						sanskritName={data.sanskritName}
						effects={data.effects}
					/>
				);
			})}
```

File: src/pages/index.js [hunk 3]
```diff
@@ -1,76 +1,27 @@
...
-	// Local state
-	const [id, setId] = useState(null);
-
-	// Global state
-	const counter = useStore(state => state.counter);
-	const decrementCounter = useStore(state => state.decrementCounter);
-	const incrementCounter = useStore(state => state.incrementCounter);
-	const setCounter = useStore(state => state.setCounter);
-
+	console.log(db.map(exercise => exercise.name));
```
Comment (JessicaLoers) [inline src/pages/index.js]: **Clean code first 🧹** 
Please remove the `console.log()`

File: src/pages/index.js [hunk 4]
```diff
@@ -1,76 +1,27 @@
...
 	return (
 		<Layout>
 			<Head>
 				<title key="title">My Project</title>
 				<meta key="description" name="description" content="This is my project" />
```
Comment (JessicaLoers) [inline src/pages/index.js:12]: I guess you missed to adapt these lines. 😉

**Review by JessicaLoers — COMMENTED** [review]
**Hi @Mersag** 
congrats to your first Pull Request. 

as a little food for thought 🍕, I have left suggestions for the`<Card / >` component (each yoga exercise) and how to render the `<Card / >` component on the home page in` index.js`

Take a look at my suggestions and try to understand and implement them. 

**If you have any questions, feel free to ask!**

File: src/pages/index.js (hunk 4 above)
Comment (Mersag) [reply src/pages/index.js:12]: Hi Jessica , 
I am a little bit confused now. Is it possible that we can meet today ? Regards , Meral

**Review by JessicaLoers — APPROVED** [review]
well done 💜 
Your first PR is ready to merge.

**Pull request merged**